src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+export const Tabs = ({ selectedTab, onTabSelected, tabs }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const { title, id } = tab;
+
+            return (
+              <li
+                key={id}
+                className={`${selectedTab.id === id && 'is-active'}`}
+                data-cy="Tab"
+              >
+                <a
+                  onClick={() => {
+                    onTabSelected(tab || tabs[0]);
```

Comment:
`tab` here will always be defined

you shouldn't call `onTabSelected` function from props if this tab is currently selected